nixos/modules/services/video/frigate.nix

```diff
@@ -166,6 +182,17 @@ in
       '';
     };
 
+    checkConfig = mkOption {
+      type = bool;
+      default = pkgs.stdenv.hostPlatform.system == pkgs.stdenv.buildPlatform.system;
```

Comment:
I think the canonical condition is `stdenv.buildPlatform.canExecute stdenv.hostPlatform`?

Reply:
TIL